api/api/examples/audio_responses.py

```diff
@@ -82,7 +82,7 @@
             "display_name": "Freesound",
             "source_url": "https://freesound.org/",
             "logo_url": None,
-            "media_count": 828,
+            "media_count": 827,
```

Comment:
Should we add ccMixter to this stats response with `media_count` = 1?

Reply:
When I added ccMixter, the audio example tests failed. I didn't have much time to look into it. Would it be okay as a separate issue? I'd like to deploy this fix today as this issue is causing a lot of 500s